src/modules/commands/common/membercount.ts

```diff
@@ -6,14 +6,16 @@ export default new Command({
   keyword: 'membercount',
   description: 'count the members of the outfit',
   help: 'Usage: `{prefix}membercount` - count the members of the outfit',
-  callback: async ({ args, reply }) => {
-    if (args.length > 0) return
+  callback: async ({ args, reply, env }) => {
+    if (args.length > 1) return reply(env.command.getHelp(env.handler))
```

Comment:
More than 1 argument should be an error message like "Error: too many arguments."